Temperature Convertor/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Temperature Converter</title>
+    <script src="TeperatureConvertor.js"></script>
+  </head>
+  <body>
+    <div>
+      <h2>Temperature Converter</h2>
+      <p>
+        <label>Fahrenheit</label>
+        <input
+          id="inputFahrenheit"
+          type="number"
+          placeholder="Fahrenheit"
+          oninput="temperatureConverter(this.value)"
+          onchange="temperatureConverter(this.value)"
+        />
```

Comment:
Do you know what `this` in `this.value` mean?

Reply:
this.value is activated and fires once the user modifies it's value, for example when we input a different value in the Fahrenheit placeholder box or when we click the up/down arrows on that box. It in this case simulates the click of a "submit" button after an input is given.

Reply:
In JavaScript, the `this` keyword refers to the object. If it's in the global scope or inside of a function the `this` keyword refers to the global object. In our case, the `this` keyword is used to refer to the element that has received the `oninput` and `onchange` events.